01-Login/src/Auth/Auth.js

```diff
@@ -1,14 +1,17 @@
-import history from '../history';
 import auth0 from 'auth0-js';
+import history from '../history';
 import { AUTH_CONFIG } from './auth0-variables';
 
 export default class Auth {
+  expires;
+  accessToken;
+
   auth0 = new auth0.WebAuth({
     domain: AUTH_CONFIG.domain,
     clientID: AUTH_CONFIG.clientId,
     redirectUri: AUTH_CONFIG.callbackUrl,
     audience: `https://${AUTH_CONFIG.domain}/userinfo`,
-    responseType: 'token id_token',
+    responseType: 'token',
     scope: 'openid'
```

Comment:
We should use 'token id_token'

Also, let's remove the audience and scope, to simplify the code and introduce less concepts. Scope is 'openid profile email' by default in Auth0.js